packages/pwa/app/components/forms/password-requirements.jsx

```diff
@@ -54,27 +54,35 @@ const PasswordRequirements = ({value}) => {
         <Stack spacing={2}>
             <PasswordRequirement isValid={pwValidations.hasMinChars}>
                 <FormattedMessage
+                    id="password_requirement.error.eight_letter_minimum"
```

Comment:
Shouldn't this be plural (`password_requirements`) since it's in the definition of the PasswordRequirements component, not PasswordRequirement component

Reply:
I've changed it to be a plural. Thanks for catching that